src/component/Registration/Signup.jsx

```diff
@@ -1,53 +1,90 @@
-import React from 'react';
-import './Signup.css'
-import NavBar from '../NavBar/NavBar'
-import Footer from '../Footer/Footer'
+import React, { useState} from 'react';
+import './Signup.css';
+import NavBar from '../NavBar/NavBar';
+import { useDispatch } from "react-redux";
+import Footer from '../Footer/Footer';
 import { Link } from 'react-router-dom';
-import Input from './Input';
+import { useNavigate } from 'react-router-dom';
+import swal from 'sweetalert';
+import axios from 'axios';
+import { setCurrentUser, signupError } from '../../slice/signupslice';
 
-
-const registrationBody = [
-    { name: 'Name' },
-    { name: 'Username' },
-    { name: 'Email' },
-    { name: 'Password' },
-    { name: 'Confirm Password' }
-]
+const url = 'http://localhost:3006/v1/signup'
 
 const Signup = () => {
+    let navigate = useNavigate ();
+    const dispatch = useDispatch();
+    const [firstName,setfirstName] = useState('');
```

Comment:
You may want to handle all this data with object

Reply:
The data are being handled with objects nw